src/components/elements/carousel/carousel.controls.tsx

```diff
@@ -0,0 +1,123 @@
+import React, { ReactNode, useCallback, useEffect, useState } from 'react'
+import styled, { css } from 'styled-components'
+import { ChevronLeft, ChevronRight } from './carousel.styles'
+import { TCarouselButtonType, TCarouselMode, TOffset } from './carousel.types'
+import useCarousel from './use-carousel'
+import { getOffsetValue } from './utils'
+import { TColor } from 'themes/theme.types'
+
+const CarouselButton = styled.button<{
+    button_type: TCarouselButtonType
+    mode: TCarouselMode
+    color?: TColor
+    offset: TOffset
+}>`
+    outline: 0;
+    cursor: pointer;
+    background-color: transparent;
+    touch-action: manipulation;
+    position: absolute;
+    z-index: 1;
+    border: 0;
+
+    :disabled {
+        opacity: 0.6;
+    }
+    path {
+        ${({ color }) =>
+            color
+                ? css`
+                      fill: ${color};
+                  `
+                : css`
+                      fill: unset;
+                  `}
+    }
```

Comment:
```suggestion
    path {${({ color }) => css`fill: ${color || 'unset'};`}
```

Reply:
Isn't it better to use **nullish coalescing operator** in here? like this: 
`path {${({ color }) => css`fill: ${color ?? 'unset'};`}`

Reply:
cool, thanks thanks guys @mitra-deriv @mohammad-rework :pray: 
applied the suggested change.